addons/website_sale/tests/test_website_editor.py

```diff
@@ -116,79 +122,121 @@ def test_extra_images_with_new_variant(self):
             )
         self.assertEqual(1, len(product_template.product_variant_ids))
 
-    def test_resequence_images(self):
+    def test_resequence_image_first(self):
```

Comment:
maybe you can add an assert to check if the `i3` became the main image of the product.

Reply:
Good idea, done :)